testproject/Assets/Tests/Runtime/MultiprocessRuntime/TestCoordinator.cs

```diff
@@ -246,17 +259,24 @@ public void InvokeFromMethodActionRpc(Action methodInfo)
     }
 
     [ClientRpc]
-    public void TriggerActionIdClientRpc(string actionId, byte[] args, ClientRpcParams clientRpcParams = default)
+    public void TriggerActionIdClientRpc(string actionId, byte[] args, bool ignorException, ClientRpcParams clientRpcParams = default)
```

Comment:
nit :P
```suggestion
    public void TriggerActionIdClientRpc(string actionId, byte[] args, bool ignoreException, ClientRpcParams clientRpcParams = default)
```

Reply:
Please take a look at [this change](https://github.com/Unity-Technologies/com.unity.netcode.gameobjects/pull/1195/commits/c8004bed8f4b89feec07653aa72b9be94ef997dd) and mark conversation as resolved if you are okay with it.